optout/src/main/java/gov/cms/ab2d/optout/gateway/S3GatewayImpl.java

```diff
@@ -3,13 +3,18 @@
 import lombok.extern.slf4j.Slf4j;
```

Comment:
The "original" version should work as is

Reply:
I updated the main README.md in order to enforce the three options for AWS setup for both the IDE and running a jar from the command line. If the developer follows these options, he can use the original version of the class without errors; therefore, I reverted to the original version of the clsss.

Reply:
I updated the main README.md in order to enforce the three options for AWS setup for both the IDE and running a jar from the command line. If the developer follows these options, he can use the original version of the class without errors; therefore, I reverted to the original version of the clsss.